packages/next/src/lib/metadata/get-metadata-route.ts

```diff
@@ -76,16 +76,35 @@ export function fillMetadataSegment(
 /**
  * Map metadata page key to the corresponding route
  *
- * static file page key:    /app/robots.txt -> /robots.xml -> /robots.txt/route
- * dynamic route page key:  /app/robots.tsx -> /robots -> /robots.txt/route
+ * static file page key:    /robots.txt -> /robots.txt/__static_metadata_file__
+ * dynamic route page key:  /robots -> /robots.txt/route
  *
  * @param page
  * @returns
  */
-export function normalizeMetadataRoute(page: string) {
+export function normalizeMetadataRoute(
+  page: string,
+  // TODO(jiwon): This is a temporary flag to exclude copying metadata files in export mode.
```

Comment:
What's the long term plan for this flag? it's tricky to have this a optional flag. inconsistent behavior can be triggered in hidden places and lead to bugs.